policy/release/lib/attestations.rego

```diff
@@ -14,7 +14,11 @@ taskrun_att_build_types := [
 	"https://tekton.dev/attestations/chains@v2",
 ]
 
-hacbs_test_task_result_name := "HACBS_TEST_OUTPUT"
+# (We can't call this test_task_result_name since anything prefixed
+# with test_ is treated as though it was a test.)
+task_test_result_name := "TEST_OUTPUT"
```

Comment:
Could we do it like this:

```suggestion
task_test_result_name := ".*TEST_OUTPUT"
```

And then change `results_named` not to test for equality but via `regex.match`? This is similar to type hinting in Chains.

Reply:
It's a reasonable idea, but I don't want to do it here. I'd rather keep this PR as minimal as possible.

Reply:
Thinking about it some more, that implementation would probably be even more minimal, but it's functionally different, e.g. (a contrived example) IGNORE_THIS_TEST_OUTPUT would be included.

Also I think we can unblock some other PRs by merging this sooner.